src/auth.js

```diff
@@ -1,14 +1,16 @@
-let isAuthorized = false
+import Cookie from 'js-cookie'
+
 export default {
-  async authorize () {
-    isAuthorized = true
-    return Promise.resolve(isAuthorized)
+  async authorize (token) {
+    const inTwentyFourHours = new Date(new Date().getTime() + 24 * 60 * 60 * 1000);
+    Cookie.set('id_token', token, {expires: inTwentyFourHours})
+    return Promise.resolve(this.isAuthorized())
```

Comment:
Why `Promise.resolve`?

Reply:
No reason, it can be taken out. But it was there and I wanted to have a small diff

Reply:
This was originally to test that the `async`/`await` stuff was working.